src/lib/sql/sql_pipeline_statement.cpp

```diff
@@ -194,8 +194,17 @@ const std::shared_ptr<const Table>& SQLPipelineStatement::get_result_table() {
                 reinterpret_cast<uintptr_t>(this));
   CurrentScheduler::schedule_and_wait_for_tasks(tasks);
 
+  if (_transaction_context && _transaction_context->aborted()) {
+    _mark_as_failed();
+    return _result_table;
```

Comment:
This means `return nullptr`, correct? Rather be explicit. Couple of lines below as well.